src/cclib/parser/data.py

```diff
@@ -43,11 +44,13 @@ class ccData(object):
         fooverlaps -- fragment orbital overlap matrix (array[2])
         fragnames -- names of fragments (list of strings)
         frags -- indices of atoms in a fragment (list of lists)
+        functional -- name of DFT functionals (string)
         gbasis -- coefficients and exponents of Gaussian basis functions (PyQuante format)
         geotargets -- targets for convergence of geometry optimization (array[1])
         geovalues -- current values for convergence of geometry optmization (array[1])
         grads -- current values of forces (gradients) in geometry optimization (array[3])
         hessian -- elements of the force constant matrix (array[1])
+        hofenergies -- heat of formation after semiempirical calculation (array[1], eV)

```

Comment:
I'll remove it for Dalton. It is for semiempirical methods and is different from ethalpy. Some packages, such as Gamess and Gaussian, have the semiempirical methods.